### DISCUSSION TITLE
Reason behind lack of min-width defaults?

### DISCUSSION THREAD

**Discussion by zkwbbr** [tailwindlabs/tailwindcss#1653] [General]
Just like to know why [min-width](https://tailwindcss.com/docs/min-width/) only has 2 default classes while [max-width](https://tailwindcss.com/docs/max-width/) seems to have way more?

**Comment by adamwathan** [comment]
What's an example of a situation where you'd use a min-width other than the provided ones? If there are good general purpose use cases for them we can definitely consider adding them. Personally I've never needed to extend that scale, but I use the max-width ones all the time.

**Reply by zkwbbr** [reply]
I have not many use case yet as I've literally started using TW 2 days ago, but fiddling around:

https://jsfiddle.net/Lxvh9bnk/

On this use case, if I want to widen the yellow `div`, my first instinct would be to use `min-width` on it. Looking around the docs, I could achieve a similar effect with using the percentage widths like `w-1/2` but I am not sure if that would be that the best alternative.
